config-stage.template

```diff
@@ -65,8 +65,7 @@
    {relay_limit, 50},
    {disable_gateway_cache, true},
    {gw_cache_retention_limit, 0},
-   {gw_context_cache_max_size, 0},
-   {blocks_to_protect_from_gc, 4000}
```

Comment:
this is removed from upstream. we either missed it or didn't save after change.

Reply:
My concern with removing is that depending on what chain variables are set and until the light gateway stuff is turned on, miners will still be maintaining blockchain state and this setting will still be important.

Reply:
I checked code, I couldn't find a reference to it in the erlang code. But I am bringing it back. If it is not used it can just stay there doing nothing. Doesn't hurt.